src/components/Capture/__tests__/Document.test.tsx

```diff
@@ -1,127 +0,0 @@
-import { h } from 'preact'
```

Comment:
Why are the `Capture/__tests__` removed?

Reply:
No idea how that happened. Correcting. 🙂